beacon-chain/p2p/service.go

```diff
@@ -478,8 +478,8 @@ func (s *Service) connectWithPeer(ctx context.Context, info peer.AddrInfo) error
 	ctx, cancel := context.WithTimeout(ctx, maxDialTimeout)
 	defer cancel()
 	if err := s.host.Connect(ctx, info); err != nil {
-		s.Peers().Scorers().BadResponsesScorer().Increment(info.ID)
-		return err
+		score := s.Peers().Scorers().BadResponsesScorer().Increment(info.ID)
+		return errors.Wrapf(err, "connect to peer %s - new bad responses score: %d", info.ID, score)
```

Comment:
Since we are using score only once would it not be better to just use the value?
```suggestion
		return errors.Wrapf(err, "connect to peer %s - new bad responses score: %d", info.ID, s.Peers().Scorers().BadResponsesScorer().Increment(info.ID))
```